options: add -h (help), show complete options at usage()

avih opened the pull request:
I spent some time initially figuring out what the options were, so this makes them more discoverable.

avih committed:
options: add -h (help), show complete options at usage()

avih commented:
Force-pushed a typo fix `Unsage` -> `Usage`.

larsbrinkhoff commented on vt100/main.c:
```diff
@@ -110,7 +110,28 @@ static int cputhread (void *arg)
+    "  -h      Display this help page and exit.\n"
+    "  -Q      Disable OpenGL and screen shader (may run faster).\n"
+    "  -N DIV  Reduce refresh-rate to 60/DIV Hz (may run faster).\n"
+    "  -2      Magnify the window. Each additional '-2' doubles the size.\n"
```
No exactly, it goes 2x, 3x, 4x, etc.

larsbrinkhoff commented on vt100/main.c:
```diff
@@ -110,7 +110,28 @@ static int cputhread (void *arg)
+    "  -c CUR  Screen curvature (0.0 - 0.5, requires OpenGL).\n"
+    "  -C      Treat Caps-Lock as Control.\n"
+    "  -B      Treat backspace as rubout (currently not implemented).\n"
+    "  -R ROM  Use ROM file instead of the built-in VT100 rom.\n"
```
-R is more like "run CP/M executable".

larsbrinkhoff commented:
This is great, thanks!  Just some minor comments.

avih commented:
> No exactly, it goes 2x, 3x, 4x, etc.

Oops.. right. Fixed.

> -R is more like "run CP/M executable".

Interesting. I updated the text to something like this, but how is this related to CP/M? as far as I can tell you just take a binary file and execute it on the 8080 after some simple setup?

avih committed:
options/usage: remove unused arguments

"b:", "r" were not used. -C is a flag (doesn't take an argument).

larsbrinkhoff commented:
> how is this related to CP/M?

It's only for testing the 8080 CPU.  There are a few test suites under the test directory.

larsbrinkhoff commented:
Your updates look good, thanks!  Merging.

avih commented:
Thanks.
